license.md

```diff
@@ -60,9 +60,9 @@ You need not contribute prototype changes, extensions, or applications that you
 
 ## Applications
 
-The purpose of this exception to [Copyleft](#copyleft) is to encourage cooperative development of this software, and to prohibit use of this software to create competing substitutes for this software that are not contributed.
+The purpose of this exception to [Copyleft](#copyleft) is to encourage cooperative development of this software by prohibiting use of this software to create competing, proprietary substitutes.
```

Comment:
Strictly speaking, the purpose described here is actually the purpose of the exception to this exception, right? I do have trouble articulating what I think better describes the purpose of this exception, though. Something something positive, collaborative community, rather than mutually destructive competition?

What do you think of something like the following? 

```suggestion
The purpose of this exception to [Copyleft](#copyleft) is to allow you to build proprietary software that uses this software, while encouraging cooperative development of this software by prohibiting use of this software to create competing, proprietary substitutes.
```

Reply:
Sorry if the email about the above comment is out of date---I edited the comment on GitHub several times after submitting it.

Reply:
> Strictly speaking, the purpose described here is actually the purpose of the exception to this exception, right?

Ouch!